### PR TITLE
[WIP]: Digitizer options

### PR DESCRIPTION
**Pull request by mwinn2**
Pull request:
- allow different Response settings Aliroot imitation vs. O2
- fix parameters for charge integration station 1 vs. 2-5
- enable charge threshold and saturation (parameter present, but not used before)
still to do: proper charge to ADC to charge conversion (parameter tuning), adding some noise, if needed

### CODE REVIEW

**Comment by mwinn2** [comment]
@aphecetche : 
Could you have a look, if things make sense (apart from the clang things, I will fix that), just to have not too many diverging code versions around.

File: Detectors/MUON/MCH/Simulation/src/Response.cxx [hunk 1]
```diff
@@ -39,6 +42,16 @@ Response::Response(Station station) : mStation(station)
...
     mK4y = 0.38312571;
     mInversePitch = 1. / 0.25; // cm^-1
+    mChargeSlope = 10.;
+    mQspreadX = 0.18;
+    mQspreadY = 0.18;
+  }
+
+  if (mSampa) {
+    mChargeThreshold = 1e-3; //arbitrary setting for the moment, was 5*1e-4
+    mChargeSat = 3000.;      // value from aliroot for charge
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Simulation/src/Response.cxx]: 3000 was the saturation value for the Manu chips which had a 12-bits ADC.
The Sampa chips having only 10-bits ADCs, that value can't be right for them. Maybe Alberto | Hervé | Christophe would know which values to use here instead ?

File: Detectors/MUON/MCH/Simulation/src/Response.cxx [hunk 2]
```diff
@@ -86,6 +99,13 @@ double Response::chargefrac1d(float min, float max, double k2, double sqrtk3, do
 double Response::response(float charge)
 {
   //FEE effects
+  if (charge < mChargeThreshold)
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Simulation/src/Response.cxx]: formatting remark here : you should but braces for the ifs even for one liners (https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Braces#Braces)

**Comment by sawenzel** [comment]
What is the status here? Is this in a state where it should be merged? Please fix formatting before.

**Comment by sawenzel** [comment]
@mwinn2 , @aphecetche  In news here?

**Comment by mwinn2** [comment]
@sawenzel:
I will finalise it today. I needed some numbers for correction.

**Comment by sawenzel** [comment]
ping

**Comment by ktf** [comment]
Please use `[WIP]` to indicate work in progress. What's the status here?

**Comment by sawenzel** [comment]
@mwinn2 : I am closing this as it is quite old without update and has merge conflicts. Please reopen when you start working on this again.

**Pull request closed without merging**